frontend/lib/models/stage_name.dart

```diff
@@ -0,0 +1,20 @@
+import 'family_name.dart';
+
+enum StageName { edge, beta, candidate, stable, proposed, updates }
+
+List<StageName> familyStages(FamilyName family) {
+  switch (family) {
+    case FamilyName.snap:
+      return [
+        StageName.edge,
+        StageName.beta,
+        StageName.candidate,
+        StageName.stable,
+      ];
+    case FamilyName.deb:
+      return [
+        StageName.proposed,
+        StageName.updates,
+      ];
+  }
+}
```

Comment:
shouldn't this be dynamic and depend on backend response?

Reply:
We don't expect these values to change. Yes we may add images later, but in that PR we can extend these values. The reason for this is that we avoid having to create endpoints for families and stages. And more importantly we avoid having to fetch them and show a spinner on the frontend before they arrive. i.e. it's just simpler. But if you still feel strongly that we should fetch families and stages I could try to work on it in a separate PR cause it will probably another big change.